### PR TITLE
Fixed the reset button bug

### PR DESCRIPTION
**Pull request by Remorax**
Fixes issue #1004 

Changes:
On clicking the reset button, the theme is reverted to what it was before the changes were made.

Demo Link: https://pr-1005-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by anshumanv** [comment]
It still seems to revert to light theme, perhaps isn't reset button meant to discard all changes in the theme and revert it back to what we started with i.e Light theme?

**Comment by Remorax** [comment]
> It still seems to revert to light theme

@anshumanv That is strange, I rechecked and it works perfectly fine for me.

>  perhaps isn't reset button meant to discard all changes in the theme and revert it back to what we started with i.e Light theme?

That's the thing, what if the user didn't start out with the light theme? What if the user prefers the dark theme, in that case, considering the light theme as the original would be wrong, no?
I for instance always use the dark theme, but when I make changes to the custom theme and press Reset, I expect it to go back to what it was before I made the changes, that is, dark theme. That's what Reset means by definition, no?

**Comment by anshumanv** [comment]
> @anshumanv That is strange, I rechecked and it works perfectly fine for me.

Not sure then, please let others check in that case. :smile:

**Comment by Remorax** [comment]
@aashutoshrathi I'll rebase it no problems. But I'm more concerned as to why you're not able to view the changes. It works perfectly fine for me, I've checked with both the accounts I have. After making the changes, when Reset button is pressed, then only the theme reverts back to the original. That is, if the theme was light originally, it reverts back to Light otherwise it reverts back to Dark. Can you check if this works for you? I'll rebase after that. Thanks 😄

**Comment by madhavrathi** [comment]
@Remorax Please check travis

**Comment by saurabhjn76** [comment]
@Remorax Please check the Travis

**Comment by Remorax** [comment]
@saurabhjn76 @madhavrathi It says that I do not have permission to deploy to the URL and that's why its' failing in 1 of 3 checks in Travis. However I have already deployed it to this very same URL: https://pr-1005-fossasia-susi-web-chat.surge.sh. So I think this PR can be merged, if its' fine otherwise. Thanks 😄

**Comment by Remorax** [comment]
@isuruAb @madhavrathi @mariobehling @saurabhjn76 Can this be merged then? Its been 23 days since I made this PR 😅

**Comment by madhavrathi** [comment]
@Remorax Please check travis errors.

**Comment by Remorax** [comment]
Its failing because it gives an error saying I do not have permission to deploy to the URL, and I think that can be ignored since I've already deployed it.
So can it be merged?@madhavrathi @rishiraj824 @saurabhjn76 
screen shot 2018-01-31 at 3 43 04 pm

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ function getStateFromStores() {
     SnackbarOpenBackground: false,
     messages: MessageStore.getAllForCurrentThread(),
     thread: ThreadStore.getCurrent(),
-		currTheme: UserPreferencesStore.getTheme(),
-		tour:true,
```
Comment (daminisatya) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: Next time make sure you don't commit the indentation changes made by your iDE. Make sure you commit only your changes.

**Pull request merged**